### DISCUSSION TITLE
用了一段时间，要求退款

### DISCUSSION THREAD

**Discussion by sekongbuer** [getlantern/lantern-forum-cn#1087] [General]
![微信图片_20241216092900](https://github.com/user-attachments/assets/4b1d05a2-93ae-46b5-8152-55c86eb5648b)
该扣多少扣多少，但请把剩下的钱退我。
以前台式机还能用，但现在台式机也连不上了。。。吐血

**Comment by MaxMKMax** [comment]
@sekongbuer 您好. 我正在帮您查看是否还可以退款吗. 一般我们建议用户用 1-2 周时就申请退款. 过了时限就不确定可不可以了. 因为支付系统会限制. 不是我们限制. 我尽力的帮您问看!

**Comment by MaxMKMax** [comment]
@sekongbuer 您也提供我您的账号邮箱. 谢谢您!

**Comment by MaxMKMax** [comment]
@sekongbuer 您好. 最近我们又更新最新版本了. 请您查看您是否是最新版本. 应该就不会有问题了. 我们也希望您别退款. 非常感谢您支持蓝灯!

**Comment by sekongbuer** [comment]
***@***.***&nbsp;


thinker
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年12月18日(星期三) 晚上11:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 用了一段时间，简直糟糕透了。要求退款 (Discussion #1087)














 
@sekongbuer 您也提供我您的账号邮箱. 谢谢您!

**Comment by sekongbuer** [comment]
刚开始还可以，但也时断时续。最后没想到这么差。基本登录不了。支持的意思是以牺牲客户用不了为代价吗？蓝灯这样会最终失去大量客户的。另外不让用户选择服务器，关键亚洲的几个基本不能用。为什么别人的vpn可以用，蓝灯不行，肯定是有问题的&nbsp;


thinker
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年12月19日(星期四) 凌晨0:38
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 用了一段时间，简直糟糕透了。要求退款 (Discussion #1087)














 
@sekongbuer 您好. 最近我们又更新最新版本了. 请您查看您是否是最新版本. 应该就不会有问题了. 我们也希望您别退款. 非常感谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@sekongbuer 您好. 您的邮箱被GitHub屏蔽了. 请您空写. xxx @ 126 .com 
我会帮您提交退款申请. 但请您要等 2周了. 美国团队要大放假了. 他们 2 周后才回来.
